ggml-backend.c

```diff
@@ -705,32 +708,83 @@ void allocate_graph(struct ggml_cgraph * gf, struct ggml_buffer * buffer) {
 
 #endif
 
-void ggml_graph_allocate_tensors(struct ggml_cgraph * graph) {
-    ggml_graph_allocate_tensors_n(&graph, 1);
+void ggml_graph_allocate_tensors(struct ggml_cgraph * graph, struct ggml_context * ctx) {
+    ggml_graph_allocate_tensors_n(&graph, 1, ctx);
 }
 
-void ggml_graph_allocate_tensors_n(struct ggml_cgraph ** graphs, int n_graphs) {
-}
+static bool ggml_is_view(struct ggml_tensor * t) {
```

Comment:
Is this name intentional? It seems confusing considering the collision with `GGML_OP_VIEW`.

Reply:
I use view generally to mean any operation that shares the memory of its parent tensor. In think the name makes sense, but I am open to suggestions.